### ISSUE TITLE
[Request]: Add Clipboard

### ISSUE THREAD

**Issue by Slackadays** [ScoopInstaller/Main#4356]
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Clipboard

### Description

Unified clipboard tool for the terminal

### Homepage

https://github.com/Slackadays/Clipboard

### Download Link(s)

https://github.com/Slackadays/Clipboard/releases/tag/0.2.1r2

### Some Indication of Popularity/Repute

Clipboard currently has 2.1K stars and 51 forks and is reviewed on console.dev

**Comment by Aziks0** [comment]
- The zip archives for versions `<=0.2.1` contain the executable at their root.
- The executable is in a `bin` directory for versions `>0.2.1`.
- The current released version on GitHub is `0.2.1r2`, but scoop consider `0.2.1` to be superior to `0.2.1r2`, [as it should](https://semver.org/#spec-item-11).

What does it mean? Well, basically, the scoop manifest will change when you release your next version:

```diff
"autoupdate": {
    "architecture": {
        "64bit": {
            "url": "https://github.com/Slackadays/Clipboard/releases/download/$version/clipboard-windows-amd64.zip",
+           "extract_dir": "bin"
        }
   }
}
```

I suggest you wait until the next version is released before adding it to scoop. Unless you're not planning to release a new version in the near future.

<details>
  <summary><code>clipboard.json</code> for version <code>0.2.1</code></summary>
  <pre><code>{
    "version": "0.2.1",
    "description": "Unified clipboard tool for the terminal",
    "homepage": "https://github.com/Slackadays/Clipboard",
    "license": "GPL-3.0-or-later",
    "architecture": {
        "64bit": {
            "url": "https://github.com/Slackadays/Clipboard/releases/download/0.2.1/clipboard-windows-amd64.zip",
            "hash": "45e5e4be41c95ae55233550e3da56c75b5db56372779c51f52f2e97b5fba977e"
        }
    },
    "bin": ["clipboard.exe", ["clipboard.exe", "cb"]],
    "checkver": {
        "github": "https://github.com/Slackadays/Clipboard"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/Slackadays/Clipboard/releases/download/$version/clipboard-windows-amd64.zip"
            }
        }
    }
}</code></pre>
</details>

**Comment by Slackadays** [comment]
Now in PR https://github.com/ScoopInstaller/Main/pull/4431
